rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigsForTest.java

```diff
@@ -66,6 +258,16 @@ public static ActivationConfig allBut(ConsensusRule... upgradesToDisable) {
         return new ActivationConfig(consensusRules);
     }
 
+    public static ActivationConfig only(List<ConsensusRule> upgradesToEnable) {
+        Map<ConsensusRule, Long> consensusRules = EnumSet.allOf(ConsensusRule.class).stream()
+                .collect(Collectors.toMap(Function.identity(), ignored -> -1L));
+        for (ConsensusRule consensusRule : upgradesToEnable) {
+            consensusRules.put(consensusRule, 0L);
+        }
+
+        return new ActivationConfig(consensusRules);
+    }
```

Comment:
```suggestion
    public static ActivationConfig only(List<ConsensusRule> upgradesToEnable) {
        return only(upgradesToEnable.toArray(new ConsensusRule[0]));
    }
```

Reply:
I will remove this overload, it is not longer used